python: can't open file '/tmp/gptme-evals-ghrov9ek/hello.py': [Errno 2] No such file or directory
